typescript/infra/scripts/utils.ts

```diff
@@ -20,6 +27,40 @@ export async function getCoreEnvironmentConfig(
   return (await import(moduleName(environment))).environment;
 }
 
+export async function getMultiProvider(
+  environment: DeployEnvironment,
+): Promise<MultiProvider<any>> {
+  return (await import(moduleName(environment))).getMultiProvider();
+}
+
+export async function getMultiProviderRemote<Networks extends ChainName>(
+  domainNames: Networks[],
+  configs: ChainMap<Networks, TransactionConfig>,
```

Comment:
we are using `domainNames` and indexing configs with this list
do we benefit from the separate parameters here?